src/aics-image-viewer/shared/utils/playControls.ts

```diff
@@ -0,0 +1,80 @@
+import { AxisName } from "../types";
+
+type PlayAxisName = AxisName | "t";
+
+const PLAY_STEP_INTERVAL = 125;
+
+export default class PlayControls {
```

Comment:
👍 I like any refactoring that separates app logic from UI.  It is interesting to consider whether this class could even live down in the volume-viewer.